todo-app/src/App.js

```diff
@@ -1,13 +1,39 @@
 import React, { Component } from 'react';
+import Task from './Task' 
+import todoItems from './todo.json';
 
-class App extends Component {
+
+export default class App extends Component {
+    state = {
+    todos: todoItems
+  }
+
+handleTaskDone = (todoID)=>{
+    this.setState({
+            todos: this.state.todos.map(todo =>{
+                if (todo.id === todoID){
+                    todo.done =! todo.done
```

Comment:
Not sure if this was a simple mistake or a misunderstanding. The bang should be alongside the value itself, not next to the equals sign. There is the case of `!=` for (loose) equality comparison, it's important to note that this is totally different.